rtrlib/rtr_mgr.h

```diff
@@ -67,28 +68,37 @@ struct rtr_mgr_group {
 	enum rtr_mgr_status status;
 };
 
+//TODO Find a nicer way todo a linked list (without writing our own)
+struct rtr_mgr_group_node {
```

Comment:
Seems that the `struct rtr_mgr_group_node` is only used internally in rtr_mgr.c if it's true it should be moved to the .c file